src/test/java/uk/gov/digital/ho/hocs/info/api/SchemaIntegrationTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.digital.ho.hocs.info.api;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.entity.ContentType;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.core.ParameterizedTypeReference;
```

Comment:
There's a few unused imports in this class, can you run an optimize imports on the file for me? (try and avoid the whole project as it will cause a load of changes I'm guessing).

Reply:
I've done this now.